### PR TITLE
ANI-11 Create MapSystem.Domain

### PR DESCRIPTION
**Pull request by famafka**
Create entities Location, Place, PlaceTransition.

### CODE REVIEW

File: AniRPG.MapSystem.Domain/Common/IEntity.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace AniRPG.MapSystem.Domain.Common
+{
+    public interface IEntity
```
Comment (aangairbender) [inline AniRPG.MapSystem.Domain/Common/IEntity.cs]: We dont need IEntity for specific MapSystem. Other systems could also use IEntity.
So we have 2 options:
1)create some global project with IEntity
2)remove IEntity interface for now

I suggest 2nd option for current version.

File: AniRPG.MapSystem.Domain/Entities/PlaceTransition.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+namespace AniRPG.MapSystem.Domain.Entities
+{
+    public class PlaceTransition : IEntity
+    {
+        public int Id { get; set; }
+        public string Message { get; set; }
+        public Place From { get; set; }
+        public Place To { get; set; }
+
```
Comment (aangairbender) [inline AniRPG.MapSystem.Domain/Entities/PlaceTransition.cs]: empty line

File: AniRPG.MapSystem.Domain/Common/IEntity.cs (hunk 1 above)
Comment (famafka) [reply AniRPG.MapSystem.Domain/Common/IEntity.cs]: IEntity deleted for now

File: AniRPG.MapSystem.Domain/Entities/PlaceTransition.cs (hunk 2 above)
Comment (famafka) [reply AniRPG.MapSystem.Domain/Entities/PlaceTransition.cs]: removed

**Review by aangairbender — APPROVED** [review]

**Pull request merged**